gtk-sys/src/gtk_glue.c

```diff
@@ -143,6 +143,10 @@ GtkTextTagTable* cast_GtkTextTagTable(void* widget) {
     return GTK_TEXT_TAG_TABLE(widget);
 }
 
+GtkTextIter* cast_GtkTextIter(void* widget) {
+    return GTK_TEXT_ITER(widget);
```

Comment:
Oh right, in C if you call any random `foo_bar(x, y, z)` the compiler assumes there exists a `int foo_bar(...)` and goes on happily. You'll get busted by the linker if the symbol is undefined but since this function isn't used in our examples, all is well. With rust+windows things are slightly different for now and that's why appveyor has caught this.